scripts/_lines_of_code.py

```diff
@@ -45,17 +45,30 @@ def execute_bind(id_, message):
 class NotContributedError(Exception):
     pass
 
-
+excepted_files = [
+    'package.json',
+    'package-lock.json',
+    '.gitignore',
+    'yarn.lock'
```

Comment:
Add LICENCE file to the list.
```suggestion
    'yarn.lock',
    'LICENSE'
```